src/services/safe_border.py

```diff
@@ -0,0 +1,164 @@
+from src.web3py.typings import Web3
+from src.typings import BlockStamp
+from src.web3py.extentions.lido_validators import LidoValidator
+from src.modules.submodules.consensus import ChainConfig
+
+DEFAULT_SHIFT = 8 # epochs ~50 min
+MAX_NEGATIVE_REBASE_SHIFT = 1536 # epochs ~6.8 days
+FAR_FUTURE_EPOCH = 2**64 - 1
+MIN_VALIDATOR_WITHDRAWABILITY_DELAY = 2**8
+EPOCHS_PER_SLASHINGS_VECTOR = 2**13
+
+class SafeBorder:
+    def __init__(self, w3: Web3) -> None:
+        self.w3 = w3
+        self.lido_contracts = w3.lido_contracts
+
+    def get_safe_border_epoch(self, is_bunker: bool, blockstamp: BlockStamp, chain_config: ChainConfig):
```

Comment:
In each method we use DEFAULT_SHIFT
Lets move it here in one place

Reply:
That would not be correct because calculation depends on the value with shift at least here
https://github.com/lidofinance/lido-oracle/blob/3167a4e3d4f0fc3a47c8b78a19139bf471280f39/src/services/safe_border.py#L43